src/jrd/Database.h

```diff
@@ -311,6 +311,9 @@ class Database : public pool_alloc<type_dbb>
 		bool incTempCacheUsage(FB_SIZE_T size);
 		void decTempCacheUsage(FB_SIZE_T size);
 
+		bool getRestoring() const { return m_restoring; }
```

Comment:
Maybe isRestoring like the other method name?

Reply:
+1

Reply:
In `GlobalObjectHolder` it is getter and setter. In `Database` it have a bit different role.